common4j/src/main/com/microsoft/identity/common/java/nativeauth/providers/responses/signup/SignUpChallengeApiResponse.kt

```diff
@@ -40,16 +40,15 @@ import java.net.HttpURLConnection
  */
 data class SignUpChallengeApiResponse(
     @Expose override var statusCode: Int,
-    @SerializedName("signup_token") val signupToken: String?,
     @Expose @SerializedName("challenge_type") val challengeType: String?,
     @SerializedName("challenge_target_label") val challengeTargetLabel: String?,
```

Comment:
I'm looking at this field, vs. line 44 and line 49. one of these is outdated and should be removed, right?

Reply:
All of these are sent by the API. The challenge_channel field contains the channel used to sent the code e.g email and the challenge_target_label field contains the actual address in an obfuscated format